### DISCUSSION TITLE
Multiple Enterprise accounts

### DISCUSSION THREAD

**Discussion by andrew-selvia** [community/community#40754] [Enterprise] [answered]
This is a feature request. While it's nice to be able to sign in to my GH Enterprise account via the iOS app, my company has multiple GHE instances. How feasible would it be to add support for multiple Enterprise accounts in the iOS app?

GitHub for iOS, Version 1.91.0 (85250348)

**Comment by ossanna16** [comment]
Hi there @andrew-selvia and welcome to our community! Thank you for the feature request!

**Comment by eliperkins** [comment, marked as answer]
Hey @andrew-selvia and folks, we've shipped functionality this week to allow you to log in with multiple accounts, like your personal and work accounts, at a the same time! [Read more about it in our Changelog post](https://github.blog/changelog/2023-08-15-log-in-with-multiple-github-accounts-on-github-mobile/).

<img width="1200" alt="image" src="https://github.com/community/community/assets/1051453/d2ba902e-2199-4625-948a-532d64a6f44e">

Download or update GitHub Mobile today from the [Apple App Store](https://apps.apple.com/us/app/github/id1477376905) or [Google Play Store](https://play.google.com/store/apps/details?id=com.github.android) to get started.

**Reply by andrew-selvia** [reply]
Thank you! 🙏🏻😄
